src/Utility/CookieCryptTrait.php

```diff
@@ -92,10 +92,10 @@ protected function _checkCipher(string $encrypt): void
     /**
      * Decrypts $value using public $type method in Security class
      *
-     * @param array|string $values Values to decrypt
+     * @param string[]|string $values Values to decrypt
      * @param string|false $mode Encryption mode
      * @param string|null $key Used as the security salt if specified.
-     * @return string|array Decrypted values
+     * @return string|string[] Decrypted values
```

Comment:
Same need to be done for return type of  `_decode()` method called by `_decrypt()` else psalm is unhappy.